From d1404e8e153c49d9c211a2fd6f0b2b2469c6c8d2 Mon Sep 17 00:00:00 2001
From: Mark Brown <broonie@linaro.org>
Date: Sat, 21 Jun 2014 11:40:40 +0100
Subject: [PATCH] spi: Remove unused variable

Reported-by: kbuild test robot
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit 840e9c354f3492259a65ae235351cde1089a409e)
(cherry picked from commit df473b0daee67eae1ebc1b76c1f5c3a1358ac7c6)
---
 drivers/spi/spi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/spi/spi.c b/drivers/spi/spi.c
index 66b61a4..1c77a00 100644
--- a/drivers/spi/spi.c
+++ b/drivers/spi/spi.c
@@ -350,7 +350,6 @@ static DEFINE_MUTEX(board_lock);
 struct spi_device *spi_alloc_device(struct spi_master *master)
 {
 	struct spi_device	*spi;
-	struct device		*dev = master->dev.parent;
 
 	if (!spi_master_get(master))
 		return NULL;
-- 
1.8.0

